python/src/aiconfig/editor/client/src/components/prompt/AddPromptButton.tsx

```diff
@@ -61,15 +61,15 @@ export default memo(function AddPromptButton({ addPrompt, getModels }: Props) {
 
   return (
     <Menu
-      position="bottom-start"
+      position="bottom"
       // Manually maintain open state to support ... expand button
       closeOnItemClick={false}
       opened={isOpen}
       onChange={setIsOpen}
     >
       <Menu.Target>
         <Tooltip label="Add prompt">
-          <ActionIcon>
+          <ActionIcon style={{ width: "100%" }}>
```

Comment:
```suggestion
          <ActionIcon w="100%">
```
nit, let's use the inline width prop. Just noting that using `style={{ .... }}` means the style prop is a new object each time this renders, so this component and children will always re-render. In this case not a big deal because I don't think ActionIcon or IconPlus are memoized anyway, just a good habit to follow